src/cuga/backend/llm/models.py

```
@@ -557,6 +571,35 @@ def _get_ssl_verify(self, model_settings: Dict[str, Any]) -> "bool | str":
+        for env_var in ("CUGA_LLM_HTTP_TIMEOUT", "LLM_HTTP_TIMEOUT"):
+            env_val = _parse_timeout(os.environ.get(env_var))
+            if env_val is not None:
+                logger.debug(f"Using {env_var} from environment: {env_val}")
+                return env_val
+
+        try:
+            toml_timeout = _parse_timeout(settings.connections.llm_http_timeout)
+            if toml_timeout is not None:
+                return toml_timeout
+        except Exception:
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)